src/components/20-molecules/accordion/index.js

```diff
@@ -23,7 +63,9 @@ class AXAAccordion extends LitElement {
 
   static get properties() {
     return {
+      refId: { type: String, defaultValue: `accordion-${createRefId()}` },
       small: { type: Boolean },
+      level: { type: String },
```

Comment:
Since it's a new property, we have the chance to make it crystal-clear: would ariaLevel be better?